pkg/resources/schema_acceptance_test.go

```diff
@@ -31,36 +97,283 @@ func TestAcc_Schema(t *testing.T) {
+					resource.TestCheckResourceAttr("snowflake_schema.test", "name", id.Name()),
+					resource.TestCheckResourceAttr("snowflake_schema.test", "database", databaseId.Name()),
+					resource.TestCheckResourceAttr("snowflake_schema.test", "with_managed_access", "false"),
+					resource.TestCheckResourceAttr("snowflake_schema.test", "is_transient", "false"),
+
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "data_retention_time_in_days", accountDataRetentionTimeInDays),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "max_data_extension_time_in_days", accountMaxDataExtensionTimeInDays),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "external_volume", accountExternalVolume),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "catalog", accountCatalog),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "replace_invalid_characters", accountReplaceInvalidCharacters),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "default_ddl_collation", accountDefaultDdlCollation),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "storage_serialization_policy", accountStorageSerializationPolicy),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "log_level", accountLogLevel),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "trace_level", accountTraceLevel),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "suspend_task_after_num_failures", accountSuspendTaskAfterNumFailures),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "task_auto_retry_attempts", accountTaskAutoRetryAttempts),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "user_task_managed_initial_warehouse_size", accountUserTaskMangedInitialWarehouseSize),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "user_task_timeout_ms", accountUserTaskTimeoutMs),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "user_task_minimum_trigger_interval_in_seconds", accountUserTaskMinimumTriggerIntervalInSeconds),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "quoted_identifiers_ignore_case", accountQuotedIdentifiersIgnoreCase),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "enable_console_output", accountEnableConsoleOutput),
+					resource.TestCheckResourceAttrPtr("snowflake_schema.test", "pipe_execution_paused", accountPipeExecutionPaused),
 				),
 			},
-			// UPDATE COMMENT (proves issue https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2606)
+			// import - without optionals
 			{
-				ConfigDirectory: config.TestNameDirectory(),
-				ConfigVariables: map[string]config.Variable{
-					"name":     config.StringVariable(name),
-					"database": config.StringVariable(acc.TestDatabaseName),
-					"comment":  config.StringVariable(""),
+				ConfigDirectory:         acc.ConfigurationDirectory("TestAcc_Schema/basic"),
```

Comment:
does import care about the cfg dir and vars?

Reply:
When I remove them, I get `Error: No value for required variable`

Reply:
okay, but these are not used for the import tests AFAIK